doc/deployment/deploy_custom.md

```diff
@@ -320,41 +295,43 @@ The Pachyderm's service account is called  `pachyderm`.
 
 ### Role or ClusterRole
 
-Depending on whether you used the `--local-roles` flag or not, the next manifest will be of `kind` `Role` or `ClusterRole`.
-depending on whether you  used the `--local-roles` flag `pachctl deploy` command.
+Depending on whether you used the `--local-roles` flag or not, the next manifest `kind` is `Role` or `ClusterRole`.
 
 ### RoleBinding or ClusterRoleBinding
 
-This manifest binds the `Rule` or `ClusterRole` to the `ServiceAccount` created above.
+This manifest binds the `Role` or `ClusterRole` to the `ServiceAccount` created above.
 
 ## Application-related
 
+This section walks you through the application-related manifests.
+
 ### PersistentVolume
 
 If you used `--static-etc-volume` to deploy Pachyderm,
 the value that you specify for `--persistent-disk` causes `pachctl` to write a manifest for creating a [`PersistentVolume`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) that Pachyderm's `etcd` uses in its [`PersistentVolumeClaim`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims).
 
-A common persistent volume uses in enterprises is an NFS mount backed by a storage fabric of some sort.
-In this case, a StorageClass for an NFS mount will be made available for consumption.
-Consult with your Kubernetes administrators to learn what resources are available for your deployment
+A common persistent volume that is used in enterprises is an NFS mount backed by a storage fabric.
+In this case, a StorageClass for an NFS mount is made available for consumption.
+Consult with your Kubernetes administrators to learn what resources are available for your deployment.
 
 ### PersistentVolumeClaim
 
-Pachyderm's `etcd` uses this `PersistentVolumeClaim` if you deployed using `--static-etc-volume`.
+If you deployed Pachyderm by using `--static-etc-volume`, the Pachyderm's `etcd` uses this `PersistentVolumeClaim`.
 See this manifest's name in the `etcd` Deployment manifest below.
```

Comment:
Should this be, "you will see this manifest's name in the `etcd` Deployment manifest below." ? Or are you iterally telling the user to go to a different section?